### ISSUE TITLE
Euler a sampler is available for use for TCD lora?

### ISSUE THREAD

**Issue by bigmover** [JettHu/ComfyUI-TCD#2]
Hi guys! I test TCD lora but  found that  Euler's method combined with TCD achieve better results than the TCD sampler. I don't know why and this is normal?

**Comment by JettHu** [comment]
Yeah, EulerA sampler can be used on TCD-LoRA, and EulerA sampler can also be used on LCM-LoRA. It gives decent results, but for better work it may require a special sampler.

[discussed](https://github.com/comfyanonymous/ComfyUI/pull/3370#issuecomment-2144719946)

**Comment by bigmover** [comment]
> Yeah, EulerA sampler can be used on TCD-LoRA, and EulerA sampler can also be used on LCM-LoRA. It gives decent results, but for better work it may require a special sampler.
> 
> [discussed](https://github.com/comfyanonymous/ComfyUI/pull/3370#issuecomment-2144719946)

Thank you for your kind reply.  Is there a good implementation of a sampler available somewhere right now? Somehow Comfyui can't get similar results to Diffusers. I think the gap come from the implementation of a sampler? Any advice about it?

**Comment by dfl** [comment]
theoretically, with eta parameter at 1.0 it should be identical to Euler?
(It is confusing in the code if the value should be 1-eta or eta... it seems the eta=1 is smooth like Euler but in the README it says eta=0 is deterministic)
